backend/src/main/java/de/neuefische/backend/model/Appointments.java

```diff
@@ -0,0 +1,35 @@
+package de.neuefische.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDateTime;
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Document("appointments")
+public class Appointments {
```

Comment:
As I understand this is one apointment so maybe use singular form here?

Reply:
You're right, will rename the class to the singular form